scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -21,11 +21,11 @@ To learn more about the GVS workflow, see the [Genomic Variant Store workflow ov
 
 Example GVCF and index files in the Data tab of the [GVS beta workspace](https://app.terra.bio/#workspaces/gvs-prod/Genomic_Variant_Store_Beta) are hosted in a public Google bucket and links are provided in the sample data table.
 
-While the GVS workflow has been tested with 100,000 single sample GVCF files as input, only datasets of up to 10,000 files are being used for beta testing.
+While the GVS has been tested with 250,000 single sample genome GVCF files as input, only datasets of up to 25,000 genomes are being used for beta testing.
```

Comment:
is it "genome GVCF" or just "GVCF"?

Reply:
At a minimum we'll want to up the ingest autoscaling to handle 25K as discussed with @koncheto-broad in Slack

Reply:
Currently just genomes supported so that's why I clarified. When I update docs for exomes I'll modify to talk about exomes.

Not sure what slack conversation you're referring to can you tag me in it?

Reply:
would "whole genome GVCF" be clearer?

Reply:
Fixed!